src/dyndep_parser.cc

```diff
@@ -95,9 +95,9 @@ bool DyndepParser::ParseDyndepVersion(string* err) {
 bool DyndepParser::ParseLet(string* key, EvalString* value, string* err) {
   if (!lexer_.ReadIdent(key))
     return lexer_.Error("expected variable name", err);
-  if (!ExpectToken(Lexer::EQUALS, err) || !lexer_.ReadVarValue(value, err))
-    return false;
-  return true;
+  if (ExpectToken(Lexer::EQUALS, err) && lexer_.ReadVarValue(value, err))
+    return true;
+  return false;
```

Comment:
can these three lines can be further simplified as a simple return statement? wdyt?

Reply:
Yeah, clang-tidy forced me to do so, too.